src/index.html

```diff
@@ -1,18 +1,66 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+      <div class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__grade">
+        <div class="card__stars stars">
```

Comment:
Add the modifier classes
```suggestion
        <div class="stars stars--4">
```